python/lsst/phosim/utils/phosim_repackager.py

```diff
@@ -127,85 +136,92 @@ def repackage(self, phosim_amp_files, out_dir='.'):
         DAY = DAYTIME[:2]
-        DAYOBS = '%s%s%s'%(YEAR, MONTH, DAY)
-        sensor[0].header['DAYOBS'] = DAYOBS
+        DAYOBS = "%s%s%s" % (YEAR, MONTH, DAY)
+        sensor[0].header["DAYOBS"] = DAYOBS
 
-        DATE = sensor[1].header['DATE']  # file creation date
-        sensor[0].header['DATE'] = DATE
-        sensor[0].header['MJD'] = astropy.time.Time(DATE, format='isot').mjd
+        DATE = sensor[1].header["DATE"]  # file creation date
+        sensor[0].header["DATE"] = DATE
+        sensor[0].header["MJD"] = astropy.time.Time(DATE, format="isot").mjd
 
-        sensor[0].header['FILTER'] = sensor[1].header['FILTER']
+        sensor[0].header["FILTER"] = sensor[1].header["FILTER"]
 
         serial = detectors.getSerial()  # eg. ITL-4400B-029
-        CCD_MANU, CCD_TYPE, CCD_NUM = serial.split('-')
-        sensor[0].header['LSST_NUM'] = serial
-        sensor[0].header['CCD_MANU'] = CCD_MANU  # eg. ITL
-        sensor[0].header['CCD_TYPE'] = CCD_TYPE  # eg. 4400B
+        CCD_MANU, CCD_TYPE, CCD_NUM = serial.split("-")
+        sensor[0].header["LSST_NUM"] = serial
+        sensor[0].header["CCD_MANU"] = CCD_MANU  # eg. ITL
+        sensor[0].header["CCD_TYPE"] = CCD_TYPE  # eg. 4400B
 
-        sensor[0].header['DETSIZE'] = noao_section_keyword(detectors.getBBox())
-        sensor[0].header['INSTRUME'] = 'lsstCam'
-        sensor[0].header['TELESCOP'] = 'LSST'
+        sensor[0].header["DETSIZE"] = noao_section_keyword(detectors.getBBox())
+        sensor[0].header["INSTRUME"] = self.instrument
+        sensor[0].header["TELESCOP"] = "LSST"
 
-        sensor[0].header['TELCODE'] = self.TELCODE
-        sensor[0].header['CONTRLLR'] = self.CONTRLLR
+        sensor[0].header["TELCODE"] = self.telcode
+        sensor[0].header["CONTRLLR"] = self.CONTRLLR
         # Set sequence number from OBSID, eg. 9006001, taking
         # the last 6 digits
-        SEQNUM = int(sensor[0].header['RUNNUM'][-6:])
-        sensor[0].header['SEQNUM'] = SEQNUM
-
-        OBSID = "%s_%s_%s_%s"%(self.TELCODE, self.CONTRLLR, DAYOBS, str(SEQNUM).zfill(6))
-        sensor[0].header['OBSID'] = OBSID
-
-        sensor[0].header['TESTTYPE'] = 'PHOSIM'
-        sensor[0].header['IMGTYPE'] = 'SKYEXP'
-        sensor[0].header['RAFTBAY'] = raft
-        sensor[0].header['CCDSLOT'] = ccdslot
-        sensor[0].header['RASTART'] = sensor[1].header['RA_DEG']
-        sensor[0].header['DECSTART'] = sensor[1].header['DEC_DEG']
-        sensor[0].header['ROTSTART'] = sensor[1].header['ROTANG']
-        sensor[0].header['RA'] = sensor[1].header['RA_DEG']
-        sensor[0].header['DEC'] = sensor[1].header['DEC_DEG']
-        sensor[0].header['ROTPA'] = sensor[1].header['ROTANG']
-        sensor[0].header['ROTCOORD'] = 'sky'
-        sensor[0].header['AMSTART'] = sensor[1].header['AIRMASS']
-        sensor[0].header['ELSTART'] = 90.-sensor[1].header['ZENITH']  # altitude angle
+        SEQNUM = int(sensor[0].header["RUNNUM"][-6:])
```

Comment:
Why it is `[-6:]`? Please consider to explain the details in the comment.

Reply:
Added explanation as in `repackage()`